packages/blocks/src/page-block/edgeless/components/rects/edgeless-selected-rect.ts

```diff
@@ -477,6 +478,7 @@ export class EdgelessSelectedRect extends WithDisposable(LitElement) {
       } else if (
         isImageBlock(element) ||
         isBookmarkBlock(element) ||
+        isAttachmentBlock(element) ||
```

Comment:
I think this judgment can be abstracted, so that it is not necessary to add a line in every place each time.

```ts
isImageBlock(element) ||
isBookmarkBlock(element) ||
isAttachmentBlock(element) ||
isEmbeddedBlock(element)
```

Reply:
@Flrande 
Done